### DISCUSSION TITLE
[multiInput] close button for token in the menu of multiInput is "clickable" even in readonly mode

### DISCUSSION THREAD

**Discussion by jiawang1** [UI5/webcomponents#9825] [General]
Hi colleague:

When I set readonly to multiInput component, the close button for the token in the input field will disappear. However, the close button for the token in the dropdown will still be visible and will react to user clicks. although it does not really delete the token from the menu.

![button](https://github.com/user-attachments/assets/4b25e2fe-4bcd-4484-bdb0-ca70a3b6a66b)

I have noticed the notes: "A read-only component can not be deleted or selected, but still provides visual feedback upon user interaction." So I did not report this as a bug. Just want to confirm is this the expected design? I think click close button but nothing happen will confuse the user.

Regards
Jay

**Comment by MarcusNotheis** [comment]
Thanks for reporting! I'll forward this discussion to our UI5 Web Components colleagues as the affected component is developed in their repository.

**Comment by nnaydenow** [comment]
Hi @jiawang1,

This seems to be an issue. I'll convert the discussion to issue. 

Thanks for the reporting!

**Comment by nnaydenow** [comment]
Issue to track: https://github.com/SAP/ui5-webcomponents/issues/9862
